.github/workflows/marketing-app-deploy.yml

```diff
@@ -131,7 +131,20 @@ jobs:
       run: |
         ruby ./deploy.rb
 
-    - name: "[Staging] Deploy Marketing App to AWS"
+    - name: "Download Built Docker Image"
+      run: |
+        docker pull ${{ needs.build-and-push-image.outputs.fullImageUri }}
+        docker create --name temp ${{ needs.build-and-push-image.outputs.fullImageUri }}
+
+    - name: "Extract Static Assets from Docker"
+      run: |
+        docker cp temp:/app/apps/marketing/.next/static/ ./static
+
+    - name: "[Test] Upload Static Assets to S3"
+      run: |
+        aws s3 cp static/ s3://cdo-marketing-static-assets-test/_next/static --recursive --quiet
```

Comment:
The `cp` command will add new files and overwrite duplicates, but it will not delete old files, risking exposing old content that we might not want public. Alternatively, `sync` would delete old files, but risks causing 404's if users are still requesting them. Are these files versioned by name/folder within the "static" directory?

Reply:
The entire bucket is set as versioned. My intention is to not delete stale assets for a period of time (90 days defined here). This is because during a deployment, a user can still request an old asset which ends up being a 404 mid-deployment. The bucket policy is set to expire noncurrent objects after 90 days.

Exposing old assets shouldn't be an issue because NextJS uses a deployment id based hash system.